### DISCUSSION TITLE
imageuriloader plugin load sprite sheet

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [rexrainbow/phaser3-rex-notes#297] [Q&A] [answered]
Hey there 👋,

I'm trying to use Phaser in React, and I'm loading the assets with the default create react app, with generates a base64 string on import.

So I need to use [your plugin (imageuriloader)](https://rexrainbow.github.io/phaser3-rex-notes/docs/site/imageuriloader/), but I don’t know you to import sprite sheets. 

I already checked your examples: https://github.com/rexrainbow/phaser3-rex-notes/tree/master/examples/imageuriloader :)

I think it should be something like this: `this.load.rexImageURI("player", Dude, { frameWidth: 32, frameHeight: 48 });`, but this way doesn’t work 🥲

Any suggestions? 😅

([If it helps my project](https://gitlab.com/201flaviosilva/ReactSchool/-/blob/master/src/Pages/Games/Projects/PhaserGame/MainScene.js#L17))

**Comment by rexrainbow** [comment, marked as answer]
Image URI loader updated, now it can [load sprite sheet](https://github.com/rexrainbow/phaser3-rex-notes/blob/master/examples/imageuriloader/load-spritesheet.js#L13-L17).
Only update minify file of this plugin, npm package will upgrade at end of this month.

**Reply by 201flaviosilva** [reply]
Ohhhhhhh ok 👌
I think I'm going blind. How did I not see this? 🤦
Thanks a lot 😁

**Reply by rexrainbow** [reply]
Well, this feature added today, from your requirement.

**Reply by 201flaviosilva** [reply]
Oooh, thank you so much (seriously) 😁
